app-ui-catalog/src/main/java/app/k9mail/ui/catalog/ui/common/list/ItemDefaultPadding.kt

```diff
@@ -1,12 +0,0 @@
-package app.k9mail.ui.catalog.ui.common.list
-
-import androidx.compose.foundation.layout.padding
-import androidx.compose.ui.Modifier
-import androidx.compose.ui.composed
-import app.k9mail.core.ui.compose.theme2.MainTheme
-
-fun Modifier.itemDefaultPadding(): Modifier = composed {
-    padding(
-        horizontal = MainTheme.spacings.double,
-    )
-}
```

Comment:
Wouldn't it be easier if we used a [composable modifier factory](https://developer.android.com/develop/ui/compose/custom-modifiers#create_a_custom_modifier_using_a_composable_modifier_factory) like this? 

```kotlin
@Composable
fun Modifier.itemDefaultPadding(): Modifier {
    return this then Modifier.padding(horizontal = MainTheme.spacings.double)
}
```

Reply:
Composed shouldn't have been used here in the first place. `PaddingValues` already provide everything we need.

The use of a modifier factory should be limited to situations where it is absolutely necessary. It's too easy to break the Modifier chain without noticing.